### PR TITLE
documenting create database options

### PR DESCRIPTION
**Pull request by nadja-muller**
Depends on https://github.com/neo-technology/neo4j/pull/9695

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala [hunk 1]
```diff
@@ -134,19 +134,35 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
...
-        "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the database already exist. " +
-        "Adding `OR REPLACE` to the command will result in any existing database being deleted and a new one created.")
-      query("CREATE DATABASE customers IF NOT EXISTS", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 0)
-      })) {}
-      query("CREATE OR REPLACE DATABASE customers", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 2)
-      })) {
-        p("This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.")
+      section("Handling Existing Databases", "administration-databases-create-database-existing", "enterprise-edition") {
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:139]: Currently these new sections does not show up in any menus, not sure if we would like that?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala [hunk 2]
```diff
@@ -134,19 +134,35 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
...
+        })) {
+          p("This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.")
+        }
+        note {
+          p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
+        }
       }
-      note {
-        p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
+      section("Options", "administration-databases-create-database-options", "enterprise-edition") {
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:155]: I do not know if there exists some sort of `clustering` flag similar to the `enterprise-edition` one. If it does it should probably be added here.

**Review by Lojjs — CHANGES_REQUESTED** [review]
I think the text you have added looks fine, but I would also add this new syntax to `Deprecations, additions and compatibility` and update the create database syntax in Glossary (see create index options for reference).

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala (hunk 2 above)
Comment (mnd999) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:155]: There does not appear to be a flare for this, just 'enterprise-edition', 'aura', 'fabric' and 'deprecated'. We could discuss adding one with the docs team if you think it is important.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala (hunk 1 above)
Comment (mnd999) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:139]: Added to the top of this file

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala (hunk 2 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:155]: I don't think it is super important if it does not exist already

**Review by Lojjs — APPROVED** [review]
I did not managed to build it locally this second time, but looking at the code changes, it looks alright now.

**Review by AlexicaWright — COMMENTED** [review]
Content-wise everything looks good, but the tests are failing.

**Comment by mnd999** [comment]
This is fixed in https://github.com/neo4j/neo4j-documentation/pull/1085, we should probably merge that

**Comment by AlexicaWright** [comment]
Please do merge #1085 if you can, then this is fine from our point of view.

**Comment by mnd999** [comment]
Tests are passing now.

**Pull request merged**